CLI create new project blazor wasm basic theme template with wrong using referencing LeptonXLiteTheme

Hi, I just created a new project with the command below and it didn't work.

`abp new ProjectTest --template app --ui blazor --pwa --database-provider ef --theme basic --connection-string "Server=DESKTOP\\SQLEXPRESS;Database=ProjectTestDB;Trusted_Connection=True" -csf -dbms SqlServer`

[11:11:08 INF] ABP CLI (https://abp.io)
[11:11:08 INF] Creating your project...
[11:11:08 INF] Project name: MaterialeShop
[11:11:08 INF] Template: app
[11:11:08 INF] Progressive Web App: yes
[11:11:08 INF] Database provider: EntityFrameworkCore
[11:11:08 INF] Connection string: Server=DESKTOP-5OTH55P\\SQLEXPRESS;Database=MaterialeShop;Trusted_Connection=True
[11:11:08 INF] DBMS: SQLServer
[11:11:08 INF] UI Framework: Blazor
[11:11:08 INF] Output folder: D:\2022_novembro2\MaterialeShop
[11:11:10 INF] Using cached template: app, version: 6.0.0
[11:11:10 INF] Theme: Basic
[11:11:10 INF] Check out the documents at https://docs.abp.io/en/abp/latest/Startup-Templates/Application
[11:11:11 INF] 'MaterialeShop' has been successfully created to 'D:\2022_novembro2\MaterialeShop'
yarn add v1.22.18
info No lockfile found.
[1/4] Resolving packages...
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > anymatch > micromatch > snapdragon > source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > anymatch > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > anymatch > micromatch > snapdragon > source-map-resolve > source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
warning @abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/aspnetcore.mvc.ui > gulp > glob-watcher > anymatch > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning "@abp/aspnetcore.mvc.ui.theme.basic > @abp/aspnetcore.mvc.ui.theme.shared > @abp/bootstrap > bootstrap@5.2.2" has unmet peer dependency "@popperjs/core@^2.11.6".
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 250 new dependencies.
info Direct dependencies
└─ @abp/aspnetcore.mvc.ui.theme.basic@5.3.5
info All dependencies
├─ @abp/aspnetcore.mvc.ui.theme.basic@5.3.5
├─ @abp/aspnetcore.mvc.ui.theme.shared@5.3.5
├─ @abp/aspnetcore.mvc.ui@5.3.5
├─ @abp/bootstrap-datepicker@5.3.5
├─ @abp/bootstrap@5.3.5
├─ @abp/datatables.net-bs5@5.3.5
├─ @abp/datatables.net@5.3.5
├─ @abp/font-awesome@5.3.5
├─ @abp/jquery-form@5.3.5
├─ @abp/jquery-validation-unobtrusive@5.3.5
├─ @abp/jquery-validation@5.3.5
├─ @abp/lodash@5.3.5
├─ @abp/luxon@5.3.5
├─ @abp/malihu-custom-scrollbar-plugin@5.3.5
├─ @abp/select2@5.3.5
├─ @abp/sweetalert2@5.3.5
├─ @abp/timeago@5.3.5
├─ @abp/toastr@5.3.5
├─ @abp/utils@5.3.5
├─ @fortawesome/fontawesome-free@5.15.4
├─ ansi-colors@4.1.3
├─ ansi-gray@0.1.1
├─ ansi-regex@2.1.1
├─ ansi-wrap@0.1.0
├─ append-buffer@1.0.2
├─ archy@1.0.0
├─ arr-filter@1.1.2
├─ array-each@1.0.1
├─ array-initial@1.1.0
├─ array-last@1.3.0
├─ array-sort@1.0.0
├─ assign-symbols@1.0.0
├─ async-each@1.0.3
├─ async-settle@1.0.0
├─ atob@2.1.2
├─ bach@1.2.0
├─ balanced-match@1.0.2
├─ base@0.11.2
├─ binary-extensions@1.13.1
├─ bootstrap-datepicker@1.9.0
├─ bootstrap@5.2.2
├─ brace-expansion@1.1.11
├─ braces@2.3.2
├─ buffer-equal@1.0.1
├─ buffer-from@1.1.2
├─ cache-base@1.0.1
├─ call-bind@1.0.2
├─ chokidar@2.1.8
├─ class-utils@0.3.6
├─ cliui@3.2.0
├─ clone-buffer@1.0.0
├─ clone-stats@1.0.0
├─ clone@2.1.2
├─ cloneable-readable@1.1.3
├─ code-point-at@1.1.0
├─ collection-map@1.0.0
├─ collection-visit@1.0.0
├─ concat-map@0.0.1
├─ concat-stream@1.6.2
├─ convert-source-map@1.9.0
├─ copy-descriptor@0.1.1
├─ copy-props@2.0.5
├─ core-util-is@1.0.3
├─ datatables.net-bs5@1.12.1
├─ datatables.net@1.12.1
├─ debug@2.6.9
├─ decamelize@1.2.0
├─ decode-uri-component@0.2.0
├─ default-compare@1.0.0
├─ default-resolution@2.0.0
├─ define-properties@1.1.4
├─ duplexify@3.7.1
├─ each-props@1.3.2
├─ end-of-stream@1.4.4
├─ error-ex@1.3.2
├─ es5-ext@0.10.62
├─ expand-brackets@2.1.4
├─ ext@1.7.0
├─ extend-object@1.0.0
├─ extglob@2.0.4
├─ fancy-log@1.3.3
├─ fast-levenshtein@1.1.4
├─ fill-range@4.0.0
├─ find-up@1.1.2
├─ findup-sync@2.0.0
├─ fined@1.2.0
├─ flagged-respawn@1.0.1
├─ flush-write-stream@1.1.1
├─ for-in@1.0.2
├─ fs-mkdirp-stream@1.0.0
├─ fs.realpath@1.0.0
├─ get-caller-file@1.0.3
├─ get-intrinsic@1.1.3
├─ get-value@2.0.6
├─ glob-stream@6.1.0
├─ glob-watcher@5.0.5
├─ glob@7.2.3
├─ global-modules@1.0.0
├─ global-prefix@1.0.2
├─ glogg@1.0.2
├─ gulp-cli@2.3.0
├─ gulp@4.0.2
├─ gulplog@1.0.0
├─ has-property-descriptors@1.0.0
├─ has-value@1.0.0
├─ hosted-git-info@2.8.9
├─ inflight@1.0.6
├─ ini@1.3.8
├─ interpret@1.4.0
├─ invert-kv@1.0.0
├─ is-accessor-descriptor@1.0.0
├─ is-arrayish@0.2.1
├─ is-binary-path@1.0.1
├─ is-core-module@2.11.0
├─ is-data-descriptor@1.0.0
├─ is-descriptor@1.0.2
├─ is-extglob@2.1.1
├─ is-fullwidth-code-point@1.0.0
├─ is-relative@1.0.0
├─ is-unc-path@1.0.0
├─ is-utf8@0.2.1
├─ is-valid-glob@1.0.0
├─ isarray@1.0.0
├─ isexe@2.0.0
├─ jquery-form@4.3.0
├─ jquery-mousewheel@3.1.13
├─ jquery-validation-unobtrusive@3.2.12
├─ jquery-validation@1.19.5
├─ jquery@3.6.1
├─ json-stable-stringify-without-jsonify@1.0.1
├─ just-compare@1.5.1
├─ just-debounce@1.1.0
├─ kind-of@3.2.2
├─ last-run@1.1.1
├─ lazystream@1.0.1
├─ lcid@1.0.0
├─ lead@1.0.0
├─ liftoff@3.1.0
├─ load-json-file@1.1.0
├─ lodash@4.17.21
├─ luxon@2.5.0
├─ malihu-custom-scrollbar-plugin@3.1.5
├─ map-cache@0.2.2
├─ map-visit@1.0.0
├─ matchdep@2.0.0
├─ merge-stream@2.0.0
├─ minimatch@3.1.2
├─ mixin-deep@1.3.2
├─ ms@2.0.0
├─ mute-stdout@1.0.1
├─ nanomatch@1.2.13
├─ next-tick@1.1.0
├─ normalize-package-data@2.5.0
├─ number-is-nan@1.0.1
├─ object-copy@0.1.0
├─ object.assign@4.1.4
├─ object.map@1.0.1
├─ object.pick@1.3.0
├─ object.reduce@1.0.1
├─ once@1.4.0
├─ ordered-read-streams@1.0.1
├─ os-locale@1.4.0
├─ parse-filepath@1.0.2
├─ parse-json@2.2.0
├─ parse-node-version@1.0.1
├─ parse-passwd@1.0.0
├─ pascalcase@0.1.1
├─ path-dirname@1.0.2
├─ path-exists@2.1.0
├─ path-parse@1.0.7
├─ path-root-regex@0.1.2
├─ path-root@0.1.1
├─ path-type@1.1.0
├─ picomatch@2.3.1
├─ pinkie@2.0.4
├─ posix-character-classes@0.1.1
├─ pretty-hrtime@1.0.3
├─ pump@2.0.1
├─ read-pkg-up@1.0.1
├─ read-pkg@1.1.0
├─ readable-stream@2.3.7
├─ readdirp@2.2.1
├─ rechoir@0.6.2
├─ remove-bom-stream@1.2.0
├─ repeat-element@1.1.4
├─ replace-ext@1.0.1
├─ replace-homedir@1.0.0
├─ require-directory@2.1.1
├─ require-main-filename@1.0.1
├─ resolve-dir@1.0.1
├─ resolve-options@1.1.0
├─ resolve-url@0.2.1
├─ resolve@1.22.1
├─ ret@0.1.15
├─ safe-buffer@5.1.2
├─ select2@4.0.13
├─ semver-greatest-satisfied-range@1.1.0
├─ semver@5.7.1
├─ set-blocking@2.0.0
├─ set-value@2.0.1
├─ snapdragon-node@2.1.1
├─ snapdragon-util@3.0.1
├─ source-map-resolve@0.5.3
├─ source-map-url@0.4.1
├─ source-map@0.5.7
├─ sparkles@1.0.1
├─ spdx-correct@3.1.1
├─ spdx-exceptions@2.3.0
├─ split-string@3.1.0
├─ stack-trace@0.0.10
├─ static-extend@0.1.2
├─ stream-exhaust@1.0.2
├─ stream-shift@1.0.1
├─ string_decoder@1.1.1
├─ strip-bom@2.0.0
├─ supports-preserve-symlinks-flag@1.0.0
├─ sver-compat@1.5.0
├─ sweetalert2@11.6.2
├─ through2-filter@3.0.0
├─ through2@2.0.5
├─ time-stamp@1.1.0
├─ timeago@1.6.7
├─ to-absolute-glob@2.0.2
├─ to-object-path@0.3.0
├─ to-regex-range@2.1.1
├─ to-through@2.0.0
├─ toastr@2.1.4
├─ type@1.2.0
├─ typedarray@0.0.6
├─ unc-path-regex@0.1.2
├─ undertaker-registry@1.0.1
├─ undertaker@1.3.0
├─ union-value@1.0.1
├─ unique-stream@2.3.1
├─ unset-value@1.0.0
├─ upath@1.2.0
├─ urix@0.1.0
├─ use@3.1.1
├─ util-deprecate@1.0.2
├─ v8flags@3.2.0
├─ validate-npm-package-license@3.0.4
├─ vinyl-fs@3.0.3
├─ vinyl-sourcemap@1.1.0
├─ which-module@1.0.0
├─ which@1.3.1
├─ wrap-ansi@2.1.0
├─ xtend@4.0.2
├─ y18n@3.2.2
├─ yargs-parser@5.0.1
└─ yargs@7.1.2
Done in 7.77s.
[11:11:19 INF] Creating initial migrations...
[11:11:26 INF] Initial migrations are created.
[11:11:26 INF] Installing client-side packages...
[11:11:26 INF] Found 1 projects.
[11:11:26 INF] D:\2022_novembro2\MaterialeShop\src\MaterialeShop.HttpApi.Host
[11:11:27 INF] Running Yarn on D:\2022_novembro2\MaterialeShop\src\MaterialeShop.HttpApi.Host
yarn install v1.22.18
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.26s.
[11:11:29 INF] Generating bundles for Blazor Wasm...
Building....:  (1/1)D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj
Executing...: dotnet build D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj
MSBuild version 17.4.0-preview-22470-08+6521b1591 for .NET
  Determining projects to restore...
  Restored D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Application.Contracts\MaterialeShop.Application.Contracts.csproj (in 534 ms).
  Restored D:\2022_novembro2\MaterialeShop\src\MaterialeShop.HttpApi.Client\MaterialeShop.HttpApi.Client.csproj (in 858 ms).
  Restored D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj (in 3,43 sec).
  1 of 4 projects are up-to-date for restore.
C:\Program Files\dotnet\sdk\7.0.100-rc.2.22477.23\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(257,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy [D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj]
C:\Program Files\dotnet\sdk\7.0.100-rc.2.22477.23\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(257,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy [D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Domain.Shared\MaterialeShop.Domain.Shared.csproj]
C:\Program Files\dotnet\sdk\7.0.100-rc.2.22477.23\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(257,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy [D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Application.Contracts\MaterialeShop.Application.Contracts.csproj]
  MaterialeShop.Domain.Shared -> D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Domain.Shared\bin\Debug\netstandard2.0\MaterialeShop.Domain.Shared.dll
  MaterialeShop.Application.Contracts -> D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Application.Contracts\bin\Debug\netstandard2.0\MaterialeShop.Application.Contracts.dll
  MaterialeShop.HttpApi.Client -> D:\2022_novembro2\MaterialeShop\src\MaterialeShop.HttpApi.Client\bin\Debug\netstandard2.0\MaterialeShop.HttpApi.Client.dll
D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShopBlazorModule.cs(11,42): error CS0234: The type or namespace name 'LeptonXLiteTheme' does not exist in the namespace 'Volo.Abp.AspNetCore.Components.Web' (are you missing an assembly reference?) [D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj]

Build FAILED.

D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShopBlazorModule.cs(11,42): error CS0234: The type or namespace name 'LeptonXLiteTheme' does not exist in the namespace 'Volo.Abp.AspNetCore.Components.Web' (are you missing an assembly reference?) [D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:04.99
Build failed for :D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\MaterialeShop.Blazor.csproj
System.Exception: Build failed!
   at Volo.Abp.Cli.Build.DefaultDotNetProjectBuilder.BuildInternal(DotNetProjectInfo project, String arguments, ConcurrentBag`1 builtProjects) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Build\DefaultDotNetProjectBuilder.cs:line 92
   at Volo.Abp.Cli.Build.DefaultDotNetProjectBuilder.BuildProjects(List`1 projects, String arguments) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Build\DefaultDotNetProjectBuilder.cs:line 41
[11:11:35 ERR] Could not load file or assembly 'D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\bin\Debug\net6.0\wwwroot\_framework\MaterialeShop.Blazor.dll'. The system cannot find the path specified.
System.IO.FileNotFoundException: Could not load file or assembly 'D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\bin\Debug\net6.0\wwwroot\_framework\MaterialeShop.Blazor.dll'. The system cannot find the path specified.
File name: 'D:\2022_novembro2\MaterialeShop\src\MaterialeShop.Blazor\bin\Debug\net6.0\wwwroot\_framework\MaterialeShop.Blazor.dll'
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, String ilPath, String niPath, ObjectHandleOnStack retAssembly)
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at Volo.Abp.Cli.Bundling.BundlingService.GetStartupModule(String assemblyPath) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Bundling\BundlingService.cs:line 280
   at Volo.Abp.Cli.Bundling.BundlingService.BundleAsync(String directory, Boolean forceBuild) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Bundling\BundlingService.cs:line 61
   at Volo.Abp.Cli.Commands.ProjectCreationCommandBase.RunBundleForBlazorWasmTemplateAsync(ProjectBuildArgs projectArgs) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Commands\ProjectCreationCommandBase.cs:line 426
   at Volo.Abp.Cli.Commands.NewCommand.ExecuteAsync(CommandLineArgs commandLineArgs) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\Commands\NewCommand.cs:line 108
   at Volo.Abp.Cli.CliService.RunInternalAsync(CommandLineArgs commandLineArgs) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\CliService.cs:line 167
   at Volo.Abp.Cli.CliService.RunAsync(String[] args) in D:\ci\Jenkins\workspace\abp-framework-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\CliService.cs:line 75


To fix it, I have to change the file **MaterialeShopBlazorModule.cs** _line 10_ to the following:

```
using Volo.Abp.AspNetCore.Components.Web.BasicTheme.Themes.Basic;
//using Volo.Abp.AspNetCore.Components.Web.LeptonXLiteTheme.Themes.LeptonXLite;
```

--- a/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/ChangeThemeStep.cs
+++ b/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/ChangeThemeStep.cs
@@ -31,9 +31,9 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
     protected void ChangeToBasicTheme(ProjectBuildContext context)
     {
         var defaultThemeName = context.BuildArgs.TemplateName is AppTemplate.TemplateName or AppNoLayersTemplate.TemplateName
-            ? "LeptonXLite" 
+            ? "LeptonXLite"
             : "LeptonX";
-        
+
         #region MVC Projects
 
         ChangeThemeToBasicForMvcProjects(context, defaultThemeName);
@@ -41,19 +41,28 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         #endregion
 
         #region MyCompanyName.MyProjectName.Blazor
-        
+
         ReplacePackageReferenceWithProjectReference(
             context,
             "/MyCompanyName.MyProjectName.Blazor/MyCompanyName.MyProjectName.Blazor.csproj",
             $"Volo.Abp.AspNetCore.Components.WebAssembly.{defaultThemeName}Theme",
             @"..\..\..\..\..\modules\basic-theme\src\Volo.Abp.AspNetCore.Components.WebAssembly.BasicTheme\Volo.Abp.AspNetCore.Components.WebAssembly.BasicTheme.csproj"
             );
-        
+
         ChangeNamespaceAndKeyword(
             context,
             "/MyCompanyName.MyProjectName.Blazor/MyProjectNameBlazorModule.cs",
             $"Volo.Abp.AspNetCore.Components.WebAssembly.{defaultThemeName}Theme",
             "Volo.Abp.AspNetCore.Components.WebAssembly.BasicTheme",
+            $"AbpAspNetCoreComponentsWebAssembly{defaultThemeName}ThemeModule",
+            "AbpAspNetCoreComponentsWebAssemblyBasicThemeModule"
+        );
+
+        ChangeNamespaceAndKeyword(
+            context,
+            "/MyCompanyName.MyProjectName.Blazor/MyProjectNameBlazorModule.cs",
+            $"Volo.Abp.AspNetCore.Components.Web.{defaultThemeName}Theme.Themes.{defaultThemeName}",
+            "Volo.Abp.AspNetCore.Components.Web.BasicTheme.Themes.Basic",
             $"AbpAspNetCoreComponentsWebAssembly{defaultThemeName}ThemeModule",
             "AbpAspNetCoreComponentsWebAssemblyBasicThemeModule"
         );
@@ -64,25 +73,25 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             $"Volo.Abp.AspNetCore.Components.Web.{defaultThemeName}Theme.Components",
             "Volo.Abp.AspNetCore.Components.Web.BasicTheme.Themes.Basic"
         );
-        
+
         #endregion
 
         #region Blazor.Server Projects
 
         ChangeThemeToBasicForBlazorProjects(context, defaultThemeName);
-        
+
         #endregion
 
         #region Angular
 
         var angularPackageName = context.BuildArgs.TemplateName is AppTemplate.TemplateName or AppNoLayersTemplate.TemplateName
-            ? "@abp/ng.theme.lepton-x" 
+            ? "@abp/ng.theme.lepton-x"
             : "@volosoft/abp.ng.theme.lepton-x";
-        
+
         ReplaceImportPackage(
-            context, 
+            context,
             "/angular/src/app/app.module.ts",
-            angularPackageName, 
+            angularPackageName,
             "@abp/ng.theme.basic"
         );
 
@@ -98,7 +107,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             "ThemeLeptonXModule",
             "ThemeBasicModule"
         );
-        
+
         RemoveLinesByStatement(
             context,
             "/angular/angular.json",
@@ -113,7 +122,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 "HttpErrorComponent, ",
                 ""
             );
-            
+
             ChangeModuleImportBetweenStatements(
                 context,
                 "/angular/src/app/app.module.ts",
@@ -128,11 +137,11 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
 
     protected void ChangeToLeptonTheme(ProjectBuildContext context)
     {
-        #region Common 
-        
+        #region Common
+
         RenameLeptonXFolders(context, folderName: "Lepton");
         AddLeptonThemeManagementReferenceToProjects(context);
-        
+
         #endregion
 
         #region MVC Projects
@@ -140,16 +149,16 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         ChangeThemeToLeptonForMvcProjects(context);
 
         #endregion
-        
+
         #region MyCompanyName.MyProjectName.Blazor
-        
+
         ReplacePackageReferenceWithProjectReference(
             context,
             "/MyCompanyName.MyProjectName.Blazor/MyCompanyName.MyProjectName.Blazor.csproj",
             "Volo.Abp.AspNetCore.Components.WebAssembly.LeptonXTheme",
             @"..\..\..\..\..\lepton-theme\src\Volo.Abp.AspNetCore.Components.WebAssembly.LeptonTheme\Volo.Abp.AspNetCore.Components.WebAssembly.LeptonTheme.csproj"
         );
-        
+
         ChangeNamespaceAndKeyword(
             context,
             "/MyCompanyName.MyProjectName.Blazor/MyProjectNameBlazorModule.cs",
@@ -168,7 +177,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
 
         #endregion
 
-        #region MyCompanyName.MyProjectName.Blazor.Server && MyCompanyName.MyProjectName.Blazor.Server.Tiered 
+        #region MyCompanyName.MyProjectName.Blazor.Server && MyCompanyName.MyProjectName.Blazor.Server.Tiered
 
         ChangeThemeToLeptonForBlazorServerProjects(context);
 
@@ -177,9 +186,9 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         #region Angular
 
         ReplaceImportPackage(
-            context, 
+            context,
             "/angular/src/app/app.module.ts",
-            "@volosoft/abp.ng.theme.lepton-x", 
+            "@volosoft/abp.ng.theme.lepton-x",
             "@volo/abp.ng.theme.lepton"
         );
 
@@ -195,7 +204,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             "ThemeLeptonXModule",
             "ThemeLeptonModule"
         );
-        
+
         RemoveLinesByStatement(
             context,
             "/angular/angular.json",
@@ -214,7 +223,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         #region MyCompanyName.MyProjectName.Blazor.Server && MyCompanyName.MyProjectName.Blazor.Server.Mongo - (app-nolayers)
 
         ChangeThemeToLeptonForNoLayersBlazorServerProjects(context);
-        
+
         #endregion
     }
 
@@ -233,7 +242,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             {
                 continue;
             }
-            
+
             AddProjectReference(file, $@"..\..\..\..\lepton-theme\src\Volo.Abp.LeptonTheme.Management.{projectName}\Volo.Abp.LeptonTheme.Management.{projectName}.csproj");
             AddModuleDependency(moduleFile, projectName, $"LeptonThemeManagement{ConvertProjectNameToModuleName($"{projectName}")}Module");
         }
@@ -241,12 +250,12 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
 
     private void ChangeThemeToLeptonForMvcProjects(ProjectBuildContext context)
     {
-        var projectNames = new[] 
+        var projectNames = new[]
         {
-            ".Web", ".HttpApi.Host", ".AuthServer", 
+            ".Web", ".HttpApi.Host", ".AuthServer",
             "" //for app-nolayers-mvc
         };
-        
+
         foreach (var projectName in projectNames)
         {
             var projectPath = $"/MyCompanyName.MyProjectName{projectName}/MyCompanyName.MyProjectName{projectName}.csproj";
@@ -255,20 +264,20 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             {
                 continue;
             }
-        
+
             var moduleFile = ConvertProjectFileToModuleFile(context, projectFile);
             if (moduleFile == null)
             {
                 continue;
             }
-        
+
             ReplacePackageReferenceWithProjectReference(
                 context,
                 projectFile.Name,
                 "Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonX",
                 @"..\..\..\..\..\lepton-theme\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.Lepton\Volo.Abp.AspNetCore.Mvc.UI.Theme.Lepton.csproj"
             );
-        
+
             ChangeNamespaceAndKeyword(
                 context,
                 moduleFile.Name,
@@ -277,7 +286,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 "LeptonXThemeBundles.Styles.Global",
                 "LeptonThemeBundles.Styles.Global"
             );
-        
+
             ChangeNamespaceAndKeyword(
                 context,
                 moduleFile.Name,
@@ -286,7 +295,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 "AbpAspNetCoreMvcUiLeptonXThemeModule",
                 "AbpAspNetCoreMvcUiLeptonThemeModule"
             );
-        
+
             RemoveLinesByStatement(
                 context,
                 moduleFile.Name,
@@ -294,10 +303,10 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             );
         }
     }
-    
+
     private void AddLeptonThemeManagementReferenceToProjects(ProjectBuildContext context)
     {
-        var projects = new Dictionary<string, string> 
+        var projects = new Dictionary<string, string>
         {
             {"Domain", "MyCompanyName.MyProjectName.Domain.csproj"},
             {"Domain.Shared", "MyCompanyName.MyProjectName.Domain.Shared.csproj"},
@@ -306,9 +315,9 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             {"HttpApi", "MyCompanyName.MyProjectName.HttpApi.csproj"},
             {"HttpApi.Client", "MyCompanyName.MyProjectName.HttpApi.Client.csproj"}
         };
-        
+
         AddUiProjectToProjects(projects, context);
-        
+
         foreach (var project in projects)
         {
             AddLeptonThemeManagementReference(context, project);
@@ -324,7 +333,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             {"HttpApi.Client", "MyCompanyName.MyProjectName.AdministrationService.HttpApi.Client.csproj"},
             {"Web", "MyCompanyName.MyProjectName.AdministrationService.Web.csproj"}
         };
-        
+
         foreach (var microserviceServiceProject in microserviceServiceProjects)
         {
             AddLeptonThemeManagementReference(context, microserviceServiceProject);
@@ -337,7 +346,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         {
             return;
         }
-        
+
         switch (context.BuildArgs.UiFramework)
         {
             case UiFramework.Mvc:
@@ -352,8 +361,8 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 break;
         }
     }
-    
-    private void AddLeptonThemeManagementReference(ProjectBuildContext context, KeyValuePair<string, string> projectInfo) 
+
+    private void AddLeptonThemeManagementReference(ProjectBuildContext context, KeyValuePair<string, string> projectInfo)
     {
         var reference = $@"..\..\..\..\..\lepton-theme\src\Volo.Abp.LeptonTheme.Management.{projectInfo.Key}\Volo.Abp.LeptonTheme.Management.{projectInfo.Key}.csproj";
         var projectFile = context.Files.FirstOrDefault(f => !f.Name.Contains("Test") && f.Name.Contains(projectInfo.Value) && f.Name.Contains(".csproj"));
@@ -367,7 +376,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         {
             return;
         }
-            
+
         AddProjectReference(projectFile, reference);
 
         AddModuleDependency(moduleFile, projectInfo.Key, $"LeptonThemeManagement{ConvertProjectNameToModuleName(projectInfo.Key)}Module",
@@ -391,12 +400,12 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 lines[i - 2] = lines[i - 2] + "," + Environment.NewLine + $"\ttypeof({dependency})";
             }
         }
-        
+
         moduleFile.SetLines(lines);
     }
 
-    protected void ReplacePackageReferenceWithProjectReference(        
-        ProjectBuildContext context, 
+    protected void ReplacePackageReferenceWithProjectReference(
+        ProjectBuildContext context,
         string targetProjectFilePath,
         string packageReference,
         string projectReference)
@@ -406,16 +415,16 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         {
             return;
         }
-      
+
         file.NormalizeLineEndings();
-        
+
         var lines = file.GetLines();
         var lineIndex = lines.FindIndex(line => line.Contains("PackageReference") && line.Contains(packageReference));
         if (lineIndex == -1)
         {
             return;
         }
-        
+
         lines[lineIndex] = lines[lineIndex].Replace(lines[lineIndex], $"\t<ProjectReference Include=\"{projectReference}\" />");
         file.SetLines(lines);
     }
@@ -498,13 +507,13 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         {
             return;
         }
-        
+
         lines[lineIndex] = lines[lineIndex].Replace(oldImportPackage, newImportPackage);
         file.SetLines(lines);
     }
 
     protected void RemoveLinesByStatement(
-        ProjectBuildContext context,        
+        ProjectBuildContext context,
         string filePath,
         string statement)
     {
@@ -515,7 +524,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         }
 
         file.NormalizeLineEndings();
-        
+
         var lines = file.GetLines();
         for (var i = 0; i < lines.Length; i++)
         {
@@ -529,7 +538,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
     }
 
     private void ChangeModuleImportBetweenStatements(
-        ProjectBuildContext context,        
+        ProjectBuildContext context,
         string filePath,
         string firstStatement,
         string lastStatement,
@@ -542,7 +551,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         }
 
         file.NormalizeLineEndings();
-        
+
         var lines = file.GetLines();
         var firstLineIndex = lines.FindIndex(line => line.Contains(firstStatement));
         var lastLineIndex = lines.FindIndex(line => line.Contains(lastStatement));
@@ -551,7 +560,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         {
             return;
         }
-        
+
         lines[firstLineIndex] = newStatement;
 
         for (var i = firstLineIndex + 1; i <= lastLineIndex; i++)
@@ -564,7 +573,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
     }
 
     protected void ReplaceMethodNames(
-        ProjectBuildContext context,        
+        ProjectBuildContext context,
         string filePath,
         string oldMethodName,
         string newMethodName)
@@ -576,7 +585,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
         }
 
         file.NormalizeLineEndings();
-        
+
         var lines = file.GetLines();
         for (var i = 0; i < lines.Length; i++)
         {
@@ -585,17 +594,17 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 lines[i] = lines[i].Replace(oldMethodName, newMethodName);
             }
         }
-        
+
         file.SetLines(lines);
     }
-    
+
     private static void AddProjectReference(FileEntry file, string reference)
     {
         if (!file.Name.Contains(".csproj"))
         {
             return;
         }
-        
+
         var doc = new XmlDocument() { PreserveWhitespace = true };
         using (var stream = StreamHelper.GenerateStreamFromString(file.Content))
         {
@@ -647,19 +656,19 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             file.SetContent(doc.OuterXml);
         }
     }
-    
+
     private static FileEntry ConvertProjectFileToModuleFile(ProjectBuildContext context, FileEntry projectFile)
     {
         var splittedProjectFileName = projectFile.Name.RemovePostFix("/").Split("/");
-        
+
         splittedProjectFileName = splittedProjectFileName.Take(splittedProjectFileName.Length - 1).ToArray();
-        
+
         var fileName = splittedProjectFileName?.Last();
         if (fileName == null)
         {
             return null;
         }
-        
+
         fileName = fileName
             .Replace("MyCompanyName.", "")
             .Replace(".csproj", "Module")
@@ -667,12 +676,12 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
 
         return context.Files.FirstOrDefault(f => f.Name.Contains(splittedProjectFileName.Last() + "/" + fileName) && f.Name.EndsWith("Module.cs"));
     }
-    
+
     private static string ConvertProjectNameToModuleName(string moduleName)
     {
         return moduleName.Replace(".", "");
     }
-    
+
     private static void RenameLeptonXFolders(ProjectBuildContext context, string folderName)
     {
         var leptonXFiles = context.Files.Where(x => x.Name.Contains("LeptonX") && x.IsDirectory);
@@ -700,7 +709,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 $"Volo.Abp.AspNetCore.Mvc.UI.Theme.{defaultThemeName}",
                 @"..\..\..\..\..\modules\basic-theme\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic\Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.csproj"
             );
-        
+
             ChangeNamespaceAndKeyword(
                 context,
                 $"/MyCompanyName.MyProjectName{project.Key}/{project.Value}.cs",
@@ -720,7 +729,7 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
             );
         }
     }
-    
+
     private void ChangeThemeToBasicForBlazorProjects(ProjectBuildContext context, string defaultThemeName)
     {
         var projects = new Dictionary<string, string>

